libs/core/tests/unit_tests/language_models/chat_models/test_cache.py

```diff
@@ -33,6 +34,36 @@ def clear(self, **kwargs: Any) -> None:
         self._cache = {}
 
 
+def test_local_cache_generate_sync() -> None:
```

Comment:
The unit tests that are testing LLM caching should be moved out of chat model caching

a good path would be 

```
unit_tests/language_models/llms/test_cache.py
```

Because the implementations of chat models and llms are distinct we also should keep the unit tests disintct